src/proxies_v8/proxies_v8.ts

```diff
@@ -271,9 +271,9 @@ proxiesV8.use('/msite/content/aggregation/search',
   proxyCreatorSunbirdSearch(express.Router(), `${CONSTANTS.KONG_API_BASE}/msite/content/aggregation/search`)
 )
 
-proxiesV8.use('/halloffame/top/learners',
+proxiesV8.use('/halloffame/top/learners/:ministryOrgId',
   // tslint:disable-next-line: max-line-length
-  proxyCreatorSunbirdSearch(express.Router(), `${CONSTANTS.KONG_API_BASE}/halloffame/top/learners`)
+  proxyCreatorSunbirdSearch(express.Router(), `${CONSTANTS.KONG_API_BASE}/halloffame/top/learners/:ministryOrgId`)
```

Comment:
Why you added :/ please check how we use for other and we already already have API basePath for /halloffame or not 
please check and confirm